config.mak

```diff
@@ -5,4 +5,4 @@ MYST_PREFIX=/opt/mystikos
 MYST_ENABLE_HOSTFS=1
 MYST_ENABLE_EXT2FS=1
 #export MYST_MEMCHECK=1
-#export MYST_RELEASE=1
+export MYST_RELEASE=1
```

Comment:
Is the intent to build Mystikos as "release build" by default?

Reply:
No, this should be removed before we merge the PR (this is just for testing the PR in the pipeline).